app/views/admin/physical_tests/_crunches_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class='row justify-content-sm-center crunches-form'>
```

Comment:
I suggest moving all the "_form" partials into their own subdir (app/views/admin/physical_tests/forms/_crunches.html.erb). This will help keep it more organized and finding stuff will be easier.